cohortextractor/backends/tpp.py

```diff
@@ -52,12 +52,22 @@ class TPPBackend(BaseBackend):
         ),
     )
 
-    practice_registrations = MappedTable(
-        source="RegistrationHistory",
+    practice_registrations = QueryTable(
```

Comment:
I initially added a QueryTable for `organisations`, but since it needs to know about the start date and end date as well as the patient id, it made more sense to add the relevant Organisation data to the practice_registrations table instead.  Otherwise we essentially build a copy of the RegistrationHistory and have to do the `date_in_range` type selections on each of them.

Reply:
Yeah, I think that's the right approach